test/DotNetIsolated60/Program.cs

```diff
@@ -11,14 +11,12 @@
 if (useProxy)
 {
     hostBuilder
-        .ConfigureFunctionsWebApplication()
-        .ConfigureGeneratedFunctionMetadataProvider();
```

Comment:
Removed this explicit extension method call in favor of the msbuild property (which does the auto registration of this methods).